build-recipes/win_Impose.iss_dummy

```diff
@@ -0,0 +1,113 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Impose"
+#define MyAppNameSimple "Impose"
+#define MyAppVersion "0.2.2"
+#define MyAppPublisher "Impose Developers"
+#define MyAppURL "https://impose.readthedocs.io"
+#define MyAppPlatform "win32"
+#define MyAppExeName "Impose.exe"
+#define MyAppDir = SourcePath + "dist\Impose\"
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application.
+; Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{9700CB56-B733-14CD-ABBC-CAC320CE7C89}
```

Comment:
We need to generate a new app id for Impose here. Otherwise it collides with the one for BMicro.